tests/unit/core/test_utils.js

```diff
@@ -1130,3 +1131,27 @@ describe('parsePage', () => {
     expect(parsePage(undefined)).toBe(1);
   });
 });
+
+describe('sanitizeUserHTML', () => {
+  const sanitize = (...args) => sanitizeUserHTML(...args).__html;
+
+  it('converts new lines to breaks', () => {
+    expect(sanitize(`
+      first
+      second
+    `).trim()).toEqual('<br>      first<br>      second<br>');
+  });
+
+  it('allows some tags', () => {
+    const customHtml =
+      '<b>check</b> <i>out</i> <a href="http://mysite">my site</a>';
+    expect(sanitize(customHtml)).toMatch(new RegExp(
```

Comment:
Again, this can probably use [`.toHaveHTML`](https://github.com/blainekasten/enzyme-matchers#tohavehtmlhtmlstring)